shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -91,20 +91,24 @@ const CodeEditor = createClass({
 			},
 			foldGutter  : true,
 			foldOptions : {
+				scanUp      : true,
```

Comment:
Everything looks good except this last bit. For some reason, Ctrl-\ still isn't quite right. It now folds any region just fine, but pressing the keys again it just folds the next block above instead of toggling back the current block. Do you have any ideas how to get the toggling to work?

Reply:
Toggling a folded section open seems to work as expected when the cursor is on the starting position of the folded segment. 

That was part of why I chose not to enable scanUp, I'll do some digging into the logic and see if it's something I can fix.

Reply:
When press the hotkey to close, it places my cursor at the end of the section rather than at the front. Also I can't seem to click the starting position of the folded segment to put my cursor there even if i wanted, since clicking anywhere in the line just unfolds it anyway.

>That was part of why I chose not to enable scanUp 

Without scanUp though, you have to manually place the cursor into position before you can hit the hotkey, which kind of defeats the purpose of having a hotkey.

Reply:
Hmm maybe we consider if it even needs to be a hotkey? There might also be some additional variables I could set in the foldCode call that could fix these issues.

Reply:
If you want, we can remove that particular hotkey for now and get this merged. Then if you want to take a closer look at it later it can be in a separate PR.

Reply:
Just removed the shortcut, I'll do some more research on it and add it at a later date.